README.md

```diff
@@ -1 +1,164 @@
-# cataloge-of-my-thing
\ No newline at end of file
+ <br/>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog of my things ](#-catalog-of-my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Presentation](#presentation)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+
+**Catalog of my things** In this project, we will create a console app that will help you to keep a record of different types of things you own: books, music albums, and games. Everything will be based on the UML class diagram. The data will be stored in JSON files but we will also prepare a database with a tables structure analogical to our program's class structure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby.
+- SQL.
+
+### Key Features <a name="key-features"></a>
+
+- Add book, music album, and game.
+- List books, music albums, games, genres, labels, authors, and sources.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Persentation: 
+[Vedio](https://drive.google.com/drive/folders/1jW4DLUa3p4LqkQ24F-b-hXW1a2ODy8NM?usp=share_link)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Download Visual Studio Code.
+- Download Git.
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+- `Clone the repo`
+```sh
+   git clone https://github.com/girma336/cataloge-of-my-thing
+```
+### Install
+
+
+- `you can run any one of these commands to install bundle on your machine:`
+```
+- bundle init
+- bundle install
+- sudo apt bundle install (for Linux)
+```
+
+### Usage
+
+- `To run the project, execute the following command:`
+ ```
+    ruby main.rb
+    test--> rspec spec spec/file_spec.rb
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Gulalai Ahmadi**
+
+- GitHub: [Gulalai-Ahmadi](https://github.com/AhmadiGu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gulalai-ahmadi/)
+
+
+👤 **Girma**
+
+- GitHub: [Girma-Tarekegn](https://github.com/girma336)
+- LinkedIn: [Girma-Tarekegn](https://www.linkedin.com/in/girma-tarekegn/)
+- Presentation: [Video](https://www.loom.com/share/1b566fa2407545f9b1a9b403ae2c374e)
+
```

Comment:
- I would suggest the next time using zoom for video because I had to download the video in order to watch it and for potential users that would be a hassle